src/graphql/resolvers/index.js

```diff
@@ -204,16 +207,32 @@ module.exports = function Resolvers (ssb) {
 
   const roomState = {
     name: null,
+    notices: null,
     members: new Map()
   }
 
   if (process.env.ROOM_ADDRESS) {
+
+    const getRoomNotices = () => {
+      return new Promise((resolve, reject) => {
+        fetch(ROOM_URL + '/notice/list' + '?encoding=json')
+          .then(res => res.json())
+          .then(res => {
+            if (res.error) return resolve(null)
+            resolve(res)
+          })
+      })
+    }
```

Comment:
:hot_pepper: If you understand promises, you will understand that `.then` returns a Promise. that means you can instead just write:

```js
    const getRoomNotices = () => fetch(ROOM_URL + '/notice/list' + '?encoding=json')
      .then(res => res.json())
      .then(res => res.error ? null : res)
      .catch(err => console.error('getRoomNotices failed:'. err)) // returns undefined

```

Reply:
This doesn't have the most graceful failure mode, but meh